src/api/lib/tasks/dev/requests.rake

```diff
@@ -199,5 +199,325 @@ namespace :dev do
 
       puts "* Request with delete action #{request.number} has been created."
     end
+
+    desc 'Copy last 10 requests from openSUSE:Factory'
+    task requests_from_open_suse_factory: :development_environment do
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      admin = User.get_default_admin
+      User.session = admin
```

Comment:
I'm not sure if rake tasks run in their own thread (probably?) but if you never reset this it stays around for at least all following rake tasks executed.